game-core/src/main/java/games/strategy/triplea/ui/UnitChooser.java

```diff
@@ -450,6 +454,10 @@ boolean hasMultipleHitPoints() {
       return hasMultipleHits;
     }
 
+    void addChangeListener(final ScrollableTextFieldListener listener) {
+      hitTexts.stream().forEach(field -> field.addChangeListener(listener));
```

Comment:
ditto on simplification to remove `stream()` and call `forEach` directly on the list.